### ISSUE TITLE
NVDA doesn't announce name of QT trees

### ISSUE THREAD

**Issue by Oreonan** [nvaccess/nvda#12418]
### Steps to reproduce:
Here is an example of bug reproduction using channels tree in TeamTalk app
- Open a recent TeamTalk version (E.G. V5.7.1 build with QT 5.15)
- When application window get focus, NVDA only announce window title
- No try to connect to a TeamTalk server, NVDA is still not announcing name of channels tree
- Finally press up or down arrow, now NVDA announce name of tree
- To complete test go to another tree (E.G. files list of TeamTalk), NVDA doesn't announce name of new tree

### Actual behavior:
- When a tree get focus NVDA announce nothing, not name, and not contain
- Note: it's really problematic when a tree contains only one item, NVDA never announce this item

### Expected behavior:
- NVDA should always announce of tree, even if it is empty or if it contains only one item

#### NVDA installed/portable/running from source:
- Installed

#### NVDA version:
- Always latest alpha

#### Windows version:
- 10.0.19043

#### Name and version of other software in use when reproducing the issue:
- TeamTalk 5.7.1

#### Does the issue still occur after restarting your computer?
- Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
- No

#### If add-ons are disabled, is your problem still occurring?
- Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
- Yes

**Comment by leonardder** [comment]
Is this a duplicate of 12476?

**Comment by Oreonan** [comment]
Yes, so we can close it and continue discution to this in #12476
